src/main/java/org/schabi/newpipe/extractor/services/soundcloud/SoundcloudStreamInfoItemExtractor.java

```diff
@@ -30,12 +30,13 @@ public long getDuration() {
 
     @Override
     public String getUploaderName() {
-        return searchResult.getObject("user").getString("username");
+        //return searchResult.getObject("user").getString("username");
+        return searchResult.getObject("track").getObject("user").getString("username");
```

Comment:
This change didn't considered that both charts and search (and others) use it, therefore, breaking some things.

Reply:
Fixed in e9156ee499bec59a67f65ca6b78462cf40d026e9.

Reply:
thanks :)